governator-core/src/main/java/com/netflix/governator/ProvisionMetricsModule.java

```diff
@@ -53,7 +66,20 @@ protected void configure() {
         this.bindListener(Matchers.any(), listener);
         requestStaticInjection(MetricsProvisionListener.class);
```

Comment:
it feels hacky to use Guice's static injection feature solely to gain the side effect of early initialization.  Is there no other way?